pkg/controller/namespace/namespace_controller_utils.go

```diff
@@ -460,7 +460,10 @@ func ServerPreferredNamespacedGroupVersionResources(discoveryClient discovery.Di
 		return results, err
 	}
 	for _, apiGroup := range serverGroupList.Groups {
-		preferredVersion := apiGroup.PreferredVersion
+		preferredVersion, err := discovery.GetPreferredVersion(apiGroup)
+		if err != nil {
+			return results, err
```

Comment:
shouldn't this continue? otherwise we lose all information after the group that encountered an error